src/components/TextInputs/styles.js

```diff
@@ -6,18 +6,17 @@ export default EStyleSheet.create({
   container: {
     flex: 1,
     flexDirection: "row",
-    justifyContent: "flex-start",
+    // justifyContent: "flex-start",
```

Comment:
@abayo-luc It is best practice to remove code you not using in your codebase vs commenting it out.